src/MFRC522.cpp

```diff
@@ -364,6 +364,38 @@ bool MFRC522::PCD_PerformSelfTest() {
 } // End PCD_PerformSelfTest()
 
 /////////////////////////////////////////////////////////////////////////////////////
+// Power control
+/////////////////////////////////////////////////////////////////////////////////////
+
+//IMPORTANT NOTE!!!!
+//Calling any other function that uses CommandReg will disable soft power down mode !!!
+
+void MFRC522::power_down()//Note : Only soft power down mode is available throught software
+{
+	byte val=PCD_ReadRegister(CommandReg); // Read state of the command register 
```

Comment:
CS: please add the spaces around the `=`
should be `byte val = PCD_ReadRegister(CommandReg); // Read state of the command register `